notebooks/404-style-transfer-webcam/404-style-transfer.ipynb

```diff
@@ -0,0 +1,462 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Style Transfer with OpenVINO™\n",
+    "\n",
+    "This notebook demonstrates style transfer with OpenVINO, using the [Style Transfer Models](https://github.com/openvinotoolkit/open_model_zoo/tree/master/models/public/fast-neural-style-mosaic-onnx) from [ONNX Model Repository](https://github.com/onnx/models), specifically [Fast Neural Style Transfer](https://github.com/onnx/models/tree/master/vision/style_transfer/fast_neural_style) model is one of the style transfer models designed to mix the content of an image with the style of another image. \n",
+    "\n",
+    "This notebook uses five pre-trained models, for the following styles: Mosaic, Rain Princess, Candy, Udnie and Pointilism. The models are from the [ONNX Model Repository](https://github.com/onnx/models) and are based on the research paper [Perceptual Losses for Real-Time Style Transfer and Super-Resolution](https://arxiv.org/abs/1603.08155) along with [Instance Normalization](https://arxiv.org/abs/1607.08022). Final part of this notebook shows live inference results from a webcam. Additionally, you can also upload a video file.\n",
+    "\n",
+    "> **NOTE**: To use this notebook with a webcam, you need to run the notebook on a computer with a webcam. If you run the notebook on a server, the webcam will not work. However, you can still do inference on a video."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Imports"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "import collections\n",
+    "import os\n",
+    "import sys\n",
+    "import time\n",
+    "\n",
+    "import cv2\n",
+    "import numpy as np\n",
+    "from pathlib import Path\n",
+    "from IPython.display import Markdown as md\n",
+    "from IPython import display\n",
+    "from ipywidgets import interactive, ToggleButtons\n",
+    "from openvino.runtime import Core\n",
+    "\n",
+    "sys.path.append(\"../utils\")\n",
+    "import notebook_utils as utils\n",
+    "from notebook_utils import download_file"
```

Comment:
You already imported utils a line above. Use it instead of a new import.

Reply:
Removed the redundant import